accounts/cachedb_test.go

```diff
@@ -394,6 +397,23 @@ func TestAccountCache_CacheDB_SyncFS2DB(t *testing.T) {
 	ma = nil
 }
 
+func TestCacheDBFilePath (t *testing.T) {
+	dir := filepath.Join("testdata", "keystore")
+	dir, _ = filepath.Abs(dir)
+	cache := newCacheDB(dir)
+
+	accs := cache.accounts()
+
+	for _, a := range accs {
+		if filepath.IsAbs(a.File) {
+			t.Errorf("wanted relative filepath (wanted basename), got: %v", a.File)
+		}
+	}
+
+	cache.close()
+	cache = nil
```

Comment:
How come explicitly setting cache to nil? Is there some problem with delayed garbage collection? That cache isn't used outside of this function so was a little surprising to see that :)